client/gutenberg/extensions/subscriptions/edit.jsx

```diff
@@ -0,0 +1,69 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { Component } from '@wordpress/element';
+import { TextControl, FormToggle } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+import { sprintf } from '@wordpress/i18n/build/index';
+
+class SubscriptionEdit extends Component {
+	render() {
+		const { attributes, className, isSelected, instanceId, setAttributes } = this.props;
+		const {
+			subscribe_placeholder,
+			subscribe_button,
+			success_message,
+			show_subscribers_total,
+		} = attributes;
+
+		if ( isSelected ) {
+			return (
+				<div className={ className }>
+					{ __( 'Show total subscribers' ) }
+					<FormToggle
+						checked={ show_subscribers_total }
+						onChange={ () => {
+							setAttributes( { show_subscribers_total: ! show_subscribers_total } );
+						} }
+					/>
+					<TextControl placeholder={ subscribe_placeholder } required onChange={ () => {} } />
+					<input type="button" disabled value={ __( 'Subscribe' ) } />
+				</div>
+			);
+		}
+
+		let subscriberCount = null;
+		if ( show_subscribers_total ) {
+			if ( 1 === Jetpack_Initial_State.getModules.subscriptions.data.subscriber_count ) {
+				subscriberCount = sprintf(
+					__( 'Join %s other subscriber' ),
+					Jetpack_Initial_State.getModules.subscriptions.data.subscriber_count
+				);
+			} else {
+				subscriberCount = sprintf(
+					__( 'Join %s other subscribers' ),
+					Jetpack_Initial_State.getModules.subscriptions.data.subscriber_count
+				);
+			}
+		}
+
+		return (
+			<div className={ className }>
```

Comment:
This bit seems to be very similar to the one when the block is selected above. Should we make it consume the same code?